docs/roadmap.md

```diff
@@ -2,21 +2,21 @@
 
 This document provides a high-level view on our product roadmap for this and upcoming quarters. It also lists roadmap items of past quarters.
 
-## Q1 2022
+## Q2 2022
 
-### Dependant Resources
+### More flexible Resource Types
 
-Some resources require inputs form other resources to be provisioned. For example, an "RDS Instance for MySQL" is needed in order to provision a `mysql` database or an "Azure Service Bus Namespace" might be needed in order to provision an `azure-service-bus-topic`. For more sophisticated setups, it might be necessary to provision underlying infrastructure which other resources will then depend on. It might also be necessary to inject common inputs into multiple resources (e.g. individual subscriptions for multiple workloads must all subscribe to the same topic). The feature will allow resource definitions to depend on resources defined by other resource definitions.
+Humanitec already provides a range of core resource types such as [`dns`](https://docs.humanitec.com/integrations/resource-types/dns), [`postgres`](https://docs.humanitec.com/integrations/resource-types/postgres) and [`volume`](https://docs.humanitec.com/integrations/resource-types/volume). Since the introduction of Resource References (previously known as Dependent Resources) a lot more can be achieved with the resources system. We will be introducing a number of new types including `tsl-cert` for modelling TLS Certificates, `k8s-namespace` to make it easier to programmatically defined the name of namespaces and additional types to model dependencies on services in other applications or hosted in other platforms.
```

Comment:
typo: "to programmatically define"

Reply:
there is another typo 
it reads: `tsl-cert`
it should read: `tls-cert`

Reply:
Will fix them both.